boards/nrf52840dk/include/arduino_iomap.h

```diff
@@ -0,0 +1,181 @@
+#define ARDUINO_SPI_D11D12D13   SPI_DEV(0)
+/* The ISP SPI bus is also connected to P1.13, P1.14, P1.15 */
+#define ARDUINO_SPI_ISP         SPI_DEV(0)
+/** @} */
+
+/**
+ * @name    Arduino's I2C buses
+ * @{
+ */
+/**
+ * @brief   The first I2C bus is where shields for the Arduino UNO/Mega expect
+ *          it
+ */
+#define ARDUINO_I2C_UNO         I2C_DEV(0)
+/** @} */
+
+/**
+ * @name   Mapping of MCU pins to Arduino pins
+ * @{
+ */
+/* P3 header on the nRF52840DK */
+#define ARDUINO_PIN_0           GPIO_PIN(1, 1)
+#define ARDUINO_PIN_1           GPIO_PIN(1, 2)
+#define ARDUINO_PIN_2           GPIO_PIN(1, 3)
+#define ARDUINO_PIN_3           GPIO_PIN(1, 4)
+#define ARDUINO_PIN_4           GPIO_PIN(1, 5)
+#define ARDUINO_PIN_5           GPIO_PIN(1, 6)
+#define ARDUINO_PIN_6           GPIO_PIN(1, 7)
+#define ARDUINO_PIN_7           GPIO_PIN(1, 8)
+
+/* P4 header on the nRF52840DK */
+#define ARDUINO_PIN_8           GPIO_PIN(1, 10)
+#define ARDUINO_PIN_9           GPIO_PIN(1, 11)
+#define ARDUINO_PIN_10          GPIO_PIN(1, 12)
+#define ARDUINO_PIN_11          GPIO_PIN(1, 13)
+#define ARDUINO_PIN_12          GPIO_PIN(1, 14)
+#define ARDUINO_PIN_13          GPIO_PIN(1, 15)
+/* SCL and SDA on this header are not available as digital pins in the
+ * Arduino world. It is P0.26 and P0.27 here, though. Also what would be
+ * AREF in the Arduino world is P0.02 here */
+
+
+/* P6 header on the nRF52840DK */
+#define ARDUINO_PIN_14          GPIO_PIN(0, 10)
+#define ARDUINO_PIN_15          GPIO_PIN(0, 9)
+#define ARDUINO_PIN_16          GPIO_PIN(0, 8)
+#define ARDUINO_PIN_17          GPIO_PIN(0, 7)
+#define ARDUINO_PIN_18          GPIO_PIN(0, 6)
+#define ARDUINO_PIN_19          GPIO_PIN(0, 5)
+#define ARDUINO_PIN_20          GPIO_PIN(0, 1)
+#define ARDUINO_PIN_21          GPIO_PIN(0, 0)
+
+/* P24 header on the nRF52840DK
+ * Note:    The header is one column short on the right (missing a column of
+ *          two 5V supply pins) and 8 columns on the left (missing D40 to D53
+ *          and two GND on the leftmost column).
+ * Warning: Where D39 is expected a GND pin is sitting instead */
+#define ARDUINO_PIN_22          GPIO_PIN(0, 11)
+#define ARDUINO_PIN_23          GPIO_PIN(0, 12)
+#define ARDUINO_PIN_24          GPIO_PIN(0, 13)
+#define ARDUINO_PIN_25          GPIO_PIN(0, 14)
+#define ARDUINO_PIN_26          GPIO_PIN(0, 15)
+#define ARDUINO_PIN_27          GPIO_PIN(0, 16)
+#define ARDUINO_PIN_28          GPIO_PIN(0, 17)
+#define ARDUINO_PIN_29          GPIO_PIN(0, 18)
+#define ARDUINO_PIN_30          GPIO_PIN(0, 19)
+#define ARDUINO_PIN_31          GPIO_PIN(0, 20)
+#define ARDUINO_PIN_32          GPIO_PIN(0, 21)
+#define ARDUINO_PIN_33          GPIO_PIN(0, 22)
+#define ARDUINO_PIN_34          GPIO_PIN(0, 23)
+#define ARDUINO_PIN_35          GPIO_PIN(0, 24)
+#define ARDUINO_PIN_36          GPIO_PIN(0, 25)
+#define ARDUINO_PIN_37          GPIO_PIN(1, 0)
+#define ARDUINO_PIN_38          GPIO_PIN(1, 9)
+
+/* P24 header on the nRF52840DK
```

Comment:
What are "P24 header" and is it correct that the nRF has them two times?

Reply:
> What are "P24 header"

![nRF52840_DK_User_Guide_20201203](https://github.com/RIOT-OS/RIOT/assets/2041729/9f8bf048-ad39-4bec-ba3d-0ccdf7146661)

the 2x9 pin header near the bottom

> is it correct that the nRF has them two times?

no, copy paste error. Good catch!